test/helm3/ct-e2e.yaml

```diff
@@ -21,6 +21,7 @@ excluded-charts:
   - generic-ingress       # D2IQ-62811
   - istio                 # D2IQ-62812
   - knative               # D2IQ-62813
+  - knative-serving
```

Comment:
(for my own understanding) why is this being excluded ?

Reply:
@takirala This could be unnecessary. I only added it because knative was there